Formula/protoc-gen-go.rb

```diff
@@ -18,9 +19,11 @@ class ProtocGenGo < Formula
   def install
     ENV["GOPATH"] = buildpath
     ENV["GO111MODULE"] = "off"
-    (buildpath/"src/github.com/golang").mkpath
-    ln_s buildpath, buildpath/"src/github.com/golang/protobuf"
-    system "go", "build", "-o", bin/"protoc-gen-go", "protoc-gen-go/main.go"
+    (buildpath/"src/github.com/golang/protobuf").install buildpath.children
+    cd "src/github.com/golang/protobuf/protoc-gen-go" do
+      system "go", "install"
+    end
```

Comment:
I think you needn't `cd` to that path before call `go install`. You can call like this `go install package_path`.

```ruby
system "go", "install", "github.com/golang/protobuf/protoc-gen-go"
```

Reply:
Thank you!
I fixed it.